### DISCUSSION TITLE
posts/2024/05/life-song-9/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#57] [General]
# posts/2024/05/life-song-9/

https://guozheng.rbind.io/posts/2024/05/life-song-9/

**Comment by ghost** [comment]
大学很美好，社会超现实！
读文学会比工学更难找工作，人工智能比较赚钱，一般的本科出来稍微有些能力的初薪都能6K-8K，其实在那些人眼里大学只有两种，一种是982、211双一流，一种是普通本科，特别势利眼，每次找工作遇到这种HR真的很无奈，狗眼看人低。
其实可以的话还是读研好找工作，事业单位最低要求是研究生以上。

**Reply by residualsun1** [reply]
其实现在也存在研究生反而不如本科生的声音哈哈，因为用人单位强调工作经验。不过你说的也很有道理，学历可能会让HR没看到我的工作经验和能力时就淘汰我了。

现实和声音都纷纷扰扰，都听一听，但最终还是要听自己内心的声音。总归饿不死的，希望我自己脑子能灵光一点，内心也能强大一些，即便未来从事了一份不景气的工作也能慢慢变得越来越好。
